.github/workflows/ci.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
                     -   description: 'Symfony 5.*'
                         php: '8.1'
                         symfony-versions: '^5.4'
+                    -   description: 'Symfony 6.*'
+                        php: '8.1'
+                        symfony-versions: '^6.3'
+                    -   description: 'Symfony 7.*'
+                        php: '8.2'
+                        symfony-versions: '^7.0'
```

Comment:
This job should be removed as we don't have a LTS version of Symfony 7 yet anyway. Symfony 7 will already be tested in the normal job for PHP 8.2 as that will be the compatible version.